src/Menu/Menu.js

```diff
@@ -72,8 +66,14 @@ export default function Menu({
   );
 }
 
+function isGroupedData(data) {
+  return data[0].hasOwnProperty('items');
+}
+
 function renderFromData(data, getItemProps) {
```

Comment:
This could be exported too.

Reply:
`renderFromData` logic is not duplicated anywhere that I can see.  It is similar but different from `getItems`.  Not sure there is anything to do here at present.